packages/lib/preprocess/src/process-queries.cjs

```diff
@@ -64,6 +64,11 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
 			return `
                 // Update external queries
                 if (import.meta?.hot) {
+					import.meta.hot.on("vite:beforeUpdate", () => {
+						// remove all prerendered queries
+						delete props.data;
```

Comment:
Why not use the svelte invalidation methods?

Reply:
because i forgot about their existence 😔

Reply:
actually I don't think they'd work, `invalidateAll` just reruns the load function on the client side, but queries are only SSR'd on the server, so `invalidateAll` just ends up fetching the same outdated data